static/js/place_template.tsx

```diff
@@ -82,15 +82,19 @@ interface ChartCategory {
 }
 
 interface ParentPlacePropsType {
+  placeType: string;
   parentPlaces: { dcid: string; name: string }[];
 }
 
 class ParentPlace extends Component<ParentPlacePropsType, {}> {
-  constructor(props) {
+  constructor(props: ParentPlacePropsType) {
     super(props);
   }
   render() {
     const num = this.props.parentPlaces.length;
+    if (this.props.placeType === "Country" && num === 1) {
```

Comment:
Can we just unlink the last parent place on line 101? We need to unlink it everywhere a continent is displayed, not just on country pages.

Reply:
Now checking this by the node type